### ISSUE TITLE
Improve load time of configuration editor

### ISSUE THREAD

**Issue by tthuem** [tthuem/FeatureIDE#95]
The current implementation calculates automatic propagation after every features. This is not needed. Instead, it is sufficient to check for satisfiability after each selection and ignore those features leading to an invalid configuration. In the end, automatic propagation of feature selection need to be done only once.

**Comment by pinnecke** [comment]
Should be solved inside branch **issue_26** see #26. Isn't it?

**Comment by skrieter** [comment]
Fixed and merged into Master.
